.github/workflows/gradle-android-instrumented-tests.yml

```diff
@@ -66,6 +66,12 @@ jobs:
         with:
           api-level: ${{ matrix.api-level }}
 
+      - name: Enable KVM group perms
+        run: |
+          echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666", OPTIONS+="static_node=kvm"' | sudo tee /etc/udev/rules.d/99-kvm4all.rules
+          sudo udevadm control --reload-rules
+          sudo udevadm trigger --name-match=kvm
```

Comment:
This is another try to fix the damn emulator tests that timeout after 1h30min.

Seems to be working again :)